automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/WebBrowserSpec.scala

```diff
@@ -1,92 +1,21 @@
 package org.broadinstitute.dsde.firecloud.test
 
-import java.io.{File, FileInputStream, FileOutputStream}
-import java.net.URL
-import java.text.SimpleDateFormat
-import java.util.UUID
-
-import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.automation.config.Credentials
 import org.broadinstitute.dsde.firecloud.api.Orchestration
-import org.broadinstitute.dsde.firecloud.config.{Config, Credentials}
+import org.broadinstitute.dsde.firecloud.config.Config
 import org.broadinstitute.dsde.firecloud.page.user.SignInPage
 import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceListPage
-import org.broadinstitute.dsde.firecloud.util.ExceptionHandling
-import org.openqa.selenium.chrome.ChromeDriverService
-import org.openqa.selenium.remote.{Augmenter, DesiredCapabilities, LocalFileDetector, RemoteWebDriver}
-import org.openqa.selenium.{OutputType, TakesScreenshot, WebDriver}
+import org.openqa.selenium.WebDriver
 import org.scalatest.Suite
 
-import scala.sys.SystemProperties
-import scala.util.Random
-
 /**
   * Base spec for writing FireCloud web browser tests.
   */
-trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogging { self: Suite =>
+trait WebBrowserSpec extends org.broadinstitute.dsde.automation.browser.WebBrowserSpec { self: Suite =>
```

Comment:
most of this moved to `org.broadinstitute.dsde.automation.browser.WebBrowserSpec`